### PR TITLE
@ #51 | should add guide how to use cert-manager along with self-signed root CA cert from teracy-dev-certs

### PR DESCRIPTION
**Pull request by hoatle**
connect #51

### CODE REVIEW

**Review by datphan — DISMISSED** [review]

**Comment by hoatle** [comment]
need @hoavt to review the grammar

File: docs/cert-manager.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+```
+
+
+
+## Use the generated key-pair from `teracy-dev-certs` to create a CA issuer
+
+- Make sure to enable and configure `teracy-dev-certs` by following https://github.com/teracyhq-incubator/teracy-dev-entry-k8s/blob/master/config_default.yaml#L50
+
+- Generate the traditional PKCS#1 key format: PKCS#8 format is generated by `teracy-dev-certs` which is
+  not working with `cert-manager`, we need to convert it to PCKS#1 format by the following similar
```
Comment (hoavt) [inline docs/cert-manager.md]: --> by running the command below:

(*following similar* is repeated much times :)

**Review by hoavt — COMMENTED** [review]
@hoatle please check my comment.

**Comment by hoatle** [comment]
@hoavt comments applied

**Review by hoavt — APPROVED** [review]

**Pull request merged**